libs/ledger-live-common/src/families/stellar/logic.ts

```diff
@@ -315,10 +313,10 @@ export const recipientAccount = async (
 
   let accountAddress = address;
 
-  const isMuxedAccount = Boolean(StellarSdk.StrKey.isValidMed25519PublicKey(address));
+  const isMuxedAccount = StrKey.isValidMed25519PublicKey(address);
```

Comment:
Why don't you remove `Boolean()` call? Don't we lose the type?

Reply:
`isValidMed25519PublicKey` returns a Boolean